neuromation/api/storage.py

```diff
@@ -693,13 +706,20 @@ def _isrecursive(pattern: str) -> bool:
     return pattern == "**"
 
 
-def _file_status_from_api(values: Dict[str, Any]) -> FileStatus:
+def _file_status_from_api(
+    base_uri: Optional[URL], values: Dict[str, Any]
+) -> FileStatus:
+    if base_uri is None:
+        uri = URL("storage://" + values["path"].lstrip("/"))
```

Comment:
What if the path contains characters `%`, `?` or `#`?

Reply:
Do you mean `%?#` in cluster or user name?
Is it allowed at all?

Reply:
Currently they are not allowed in cluster name and user name. But user can create files with arbitrary names.

Reply:
Got you.
@serhiy-storchaka please review the updated version